ipaclient/plugins/vault.py

```diff
@@ -304,7 +279,8 @@ def forward(self, *args, **options):
                 password = password.rstrip('\n')
 
             else:
-                password = get_new_password()
+                password = self.api.Backend.textui.prompt_password(
```

Comment:
LGTM was for complete review. I tried the patch and worked for me, so add ACK and LGTM.